index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    
+    
+    <header>
+        <div class="time">
+            <img src="images/Time.png" alt="hamburger">
+            <img src="images/Group 3.png" alt="hamburger">
+        </div>
```

Comment:
[OPTIONAL]

- Kindly remove this part from your webpage as this design is just an illustration to let you visualize the mobile  view of your portfolio on the design

![image](https://github.com/ausiemass/portfolio-austin/assets/63473584/a694eab8-4495-4918-8b74-74d821915599)